packages/start-plugin-core/tests/importProtection/utils.test.ts

```
@@ -1,21 +1,26 @@
   isInsideDirectory,
+  isFileExcluded,
   normalizeFilePath,
   relativizePath,
   shouldDeferViolation,
   stripQuery,
   stripQueryAndHash,
   withoutKnownExtension,
-} from '../../src/import-protection-plugin/utils'
+} from '../../src/import-protection/utils'
+import { compileMatchers } from '../../src/import-protection/matchers'
+import { formatViolation } from '../../src/import-protection/trace'
```

ESLint

[error] 'formatViolation' is defined but never used.

(unused-imports/no-unused-imports)